tdrs-backend/tdpservice/scheduling/parser_task.py

```diff
@@ -16,6 +16,6 @@ def parse(data_file_id):
     # for undetermined amount of time.
     data_file = DataFile.objects.get(id=data_file_id)
 
-    logger.info(f"DataFile parsing started for file {data_file.filename}")
+    logger.info(f"DataFile parsing started for file {data_file.id}")
     errors = parse_datafile(data_file)
-    logger.info(f"DataFile parsing finished with {len(errors)} errors: {errors}")
+    logger.info(f"DataFile parsing finished for file {data_file.id} with {len(errors)} errors: {errors}")
```

Comment:
can we change the `__str__` or `__repr__` on `ParserError` to be more useful? like andrew said, it's a lot of random numbers that don't say a lot